ForageSDK.podspec

```diff
@@ -1,18 +1,18 @@
 Pod::Spec.new do |spec|
 
   spec.name         = "ForageSDK"
-  spec.version      = "4.4.2"
```

Comment:
There are 2 other locations where we update the version number:
- The README.md
- ForageSDK.swift

Reply:
Just pushed commits updating it in those places as well 👍🏻